src/Nest/Document/Multiple/ScrollAll/ScrollAllObservable.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Elasticsearch.Net;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace Nest
+{
+	public class ScrollAllObservable<T> : IDisposable, IObservable<IScrollAllResponse<T>> where T : class
+	{
+		private readonly IScrollAllRequest _scrollAllRequest;
+		private readonly ISearchRequest _searchRequest;
+		private readonly IConnectionSettingsValues _connectionSettings;
+		private readonly IElasticClient _client;
+		private static IList<ISort> DocOrderSort = new ReadOnlyCollection<ISort>(new List<ISort> { new SortField { Field = "_doc" } });
```

Comment:
Should `DocOrderSort` be applied **only** if a sort is not specified on the search request? If it will always be applied, I think it should also be mentioned in the comment on `Search` on `IScrollAllRequest`